src/app/services/sms/sms.service.ts

```diff
@@ -212,7 +216,17 @@ const sendSms = (
     },
   )
     .andThen(({ status, sid, errorCode, errorMessage }) => {
+      const ddTags = {
```

Comment:
should we extract these tags to a centralized location? I'm wary that duplicate tags may occur down the road, resulting in fragmented logs

Reply:
> should we extract these tags to a centralized location?

Probably not, tags are specific to a chunk of code we want to instrument. I expect that as we instrument different part of the codebase, we will get different tags, and that's OK. If we see a need for uniformalizxation, we can refactor later.

Reply:
That said, since, the same tag shape is used in 2 places, I've made a shared type definition for it.
https://github.com/opengovsg/FormSG/pull/3824/commits/91634045838ad35a7fd05457906855cc2a75b1d0

Still, because these are different cases (async response vs webhook callback), it's entirely possible we track different tags as needed.